File: backend/customerservice/src/main/java/com/etiya/customerservice/dto/IndividualCustomer/response/SearchIndividualCustomerResponseDto.java
package com.etiya.customerservice.dto.IndividualCustomer.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class SearchIndividualCustomerResponseDto {
    private Integer customerId;
    private Long nationalityId;
    private String accountNumber;
    private String firstName;
    private String middleName;
    private String lastName;
    private String mobilePhone;
}

File: frontend/src/app/features/customer/models/response/SearchIndividualCustomerResponseDto.ts
export interface SearchIndividualCustomerResponseDto {
  customerId: number;
  nationalityId: number;
  accountNumber: string;
  firstName: string;
  middleName: string;
  lastName: string;
  mobilePhone: string;
  contacts?: { gsmNumber: string }[]; // contacts alanını ekleyin
}
